bootstrap

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/env sh
```

Comment:
This has nothing to do with this change, and doesn't have anything to do with RISC-V either. Can you submit this upstream instead?

Reply:
Sorry, this is mistaken added. I will remove it.